lib/order.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'menu'
+
+class Order
+  attr_reader :orders
+  def initialize
+    @orders = []
+    @hash = Hash.new
+  end
+
+  def greet
+    puts "Thank you for visiting our takeaway website."
+    puts "please take a look at our menu."
+  end
+
+  def present(menu)
+    menu.show
+  end
+  #
+  def choose_dish
+    puts "What would you like to order? Please enter a menu item number."
+    @menu_num = gets.chomp.to_i
+  end
+  #
+  def choose_how_many(menu)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `menu`. If it's necessary, use `_` or `_menu` as an argument name to indicate that it won't be used. You can also write as `choose_how_many(*)` if you want the method to accept any arguments but don't care about them.